dietpi/dietpi-config

```diff
@@ -335,9 +335,9 @@
 
 			fi
 
-			G_WHIP_MENU_ARRAY+=("4" "Rotation (HDMI)       : $rotation_hdmi_current")
-			G_WHIP_MENU_ARRAY+=("5" "Rotation (LCD)        : $rotation_lcd_current")
-			G_WHIP_MENU_ARRAY+=("6" "Overscan              : $overscan_text")
+			G_WHIP_MENU_ARRAY+=('4' "Rotation (HDMI)       : $rotation_hdmi_current")
```

Comment:
Just for reference, with integer, we know its value and it conforms to not requiring a string.

eg: it works without ```''```
```
G_WHIP_MENU_ARRAY+=(4 "Rotation (HDMI)       : $rotation_hdmi_current")
```

Whether this would speed up the init of ```G_WHIP_MENU_ARRAY```? I have no idea lol 👍

Reply:
AFAIK in sh/bash there is no real difference between strings and integers. This is just relevant within arithmetic expression, where the "string" is (internally) checked for being an integer.

Quotes should be actually only needed, if the string/value we want to handle contains spaces. As only within single quotes, additionally no special characters are handled, it should be theoretically fastest (Without any quotes the value somehow needs to be additionally checked for `$`, `*` etc. ). But yeah I doubt that being the case for single characters 😆. Time for the next performance test 😋.